practice-app/libs/create-exercise.js

```diff
@@ -69,10 +69,12 @@ function insertAnswers(answers) {
 
 categoryID = 'Q42889';
 
-async function createExercise(wordArray) {
+async function createExercise(wordArray, nuQue=4, nuOpt=4) {
```

Comment:
We can do it the assignment inside of the function not in the definition function.
You should do "nuQue=4" and " nuOpt=4" inside the function

Reply:
i put it because if we don't send parameters about number of questions and answers, they are default 4